package.json

```diff
@@ -364,6 +373,21 @@
           "when": "view == javaProjectExplorer && viewItem =~ /java:project(?=.*?\\b\\+java\\b)(?=.*?\\b\\+uri\\b)/",
           "group": "new@20"
         },
+        {
+          "command": "java.view.package.moveFileToTrash",
+          "when": "view == javaProjectExplorer && viewItem =~ /java:(package|packageRoot)(?=.*?\\b\\+source\\b)(?=.*?\\b\\+uri\\b)/",
+          "group": "delete@10"
```

Comment:
Since we are having more and more items in the menu, I think this could be a good chance to setup some rules to group different commands in the menu.

Regarding to this, I think we can learn from what VS Code has done for the `File Explorer`, I suggest we group the delete command into `7_modification`. See: 
- https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups
- https://github.com/microsoft/vscode/blob/70e95ce5b9407c0df82d2c272f9730db1c307350/src/vs/workbench/contrib/files/browser/fileActions.contribution.ts#L540-L553

Reply:
I'll have a try to see if there is a good way to implement.